.github/workflows/release.yml

```diff
@@ -20,30 +20,10 @@ jobs:
         uses: Brightspace/third-party-actions@actions/setup-node
       - name: Install dependencies
         run: npm install
-      - name: Build
-        run: npm run browserify
       - name: Semantic Release
         id: semantic-release
         uses: BrightspaceUI/actions/semantic-release@master
         with:
           GITHUB_TOKEN: ${{ secrets.D2L_GITHUB_TOKEN }}
           NPM: true
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - name: Assume role
-        if: steps.semantic-release.outputs.VERSION != ''
-        uses: Brightspace/third-party-actions@aws-actions/configure-aws-credentials
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
-          role-to-assume: "arn:aws:iam::771734770799:role/cdn-infrastructure-ifrau"
```

Comment:
Once this merges, some cleanup can happen here:
https://github.com/Brightspace/iam-build-tokens/blob/master/terraform/roles/ifrau.tf

Reply:
I'll open up a PR to clean this up as well. :)